test/test-api.js

```diff
@@ -244,21 +245,11 @@ var launchTests = function () {
   });
 };
 
-// FIXME: is there a good way to make a server get a port for testing?
 var server;
-var portNumber = 8000;
```

Comment:
Nit: Please keep a `var portNumber;` or `var portNumber = 0;` here instead of relying on the `portNumber = ...` above to create an accidental global.